tests/unit/CountItems/utils/utils.js

```diff
@@ -23,6 +24,7 @@ describe('CountItems::utils::consolidateDataMetrics', () => {
             _currentRestoring: 0,
             _nonCurrentRestored: 0,
             _nonCurrentRestoring: 0,
+            _incompleteMPUParts: 0,
```

Comment:
as implemented _today_ (see my other comment), this is not the count of `parts` but the count of `uploads`, so the field should be named `_incompleteMPUUploads`. Or may be better instead to actually count parts.

Reply:
Counting parts does not make sense from a client point of view, as they cannot control, most of the time, how the MPUs are splited. What is important is knowing how many MPUs are incomplete and how much data is occupied by these MPUs. It's anyway not important here, because our only use case are the quotas (only on storage bytes) and reflecting the current usage in the UI (no count of objects here as well). We can however consider it for Scuba, in a separate work.